learn/advanced/pagination.md

```diff
@@ -0,0 +1,117 @@
+  // we have at least one more page with 1 result in it
+  document.querySelector("#next_button").disabled = false;
+}
+```
+
+## Not recommended: Page selection
+
+As its name states, this type of pagination is made of a numbered list of pages accompanied by next and previous buttons.
+
+![placeholder page selection UI](https://vuejsexamples.com/content/images/2018/11/vue-pagination.gif)
+
+This is is a common UI pattern that offers users a significant amount of precision when navigating results.
+
+### Implementation
+
+The general implementation of a page selection interface is similar to our [recommended solution](#recommended-previous-and-next-buttons), but you must also display a page list.
+
+To create the page list, however, you must know the exact number of total results. For example, if you have 100 results and your search result pages contain 10 results each, your selector must show a list of numbers going from 1 to 10.
+
+Since Meilisearch can only give you an estimate of total search results, it can be difficult to implement page selectors when using Meilisearch.
+
+We recommend two different workarounds to create this kind of pagination interface with Meilisearch.
+
+#### Set a hard limit of results during your search
+
+By default, a search request returns 20 search results. You can change this value to a much higher number and treat it as the effective maximum of search results you will show a user. Doing so means the size of your `hits` array is the exact number of search results you have to paginate.
+
+For example, if you set `limit` to 300, every search request made to Meilisearch returns at most 300 documents. If a query returns a `hits` array with 200 items and you want each page to display 20 results, you can create a page selector with 10 pages.
+
+// perhaps a code sample here?
+
+This method provides control and reliability. It also prevents users from paginating too far, which avoids performance loss. However, this limits the number of results your users can see.
+
+::: note
+We do not recommend caution when setting high values for `limit` as it can negatively impact performance.
+:::
+
+#### Accept the unreliable number
+
+You use `estimatedTotalHits` to calculate the number of search result pages. This means your page count is likely to change until Meilisearch retrives the last search result.
+
+For example, a query's `estimatedTotalHits` might be `100` when you fetch the first page of search results. If you are showing 20 results per page, this means your interface will display a page selector with 5 pages. When you fetch the fifth and last page according to Meilisearch's current estimate, however, `estimatedTotalHits` might change to `300`. Your page list, previously displaying 5 pages, must now show 10 total pages.
+
+This method gives users access to all search results. However, it also results in an interface that might feel unreliable due to constant and unpredictable changes.
```

Comment:
Personally, I don't think it's a good idea to include this section. We know that it provides a jarring, unreliable interface for users. I don't really see a benefit to detailing it as an option. But always happy to be convinced or, in this case, overruled ✌🏻